console/src/app/pages/instance-settings/instance-settings.component.ts

```diff
@@ -80,7 +84,10 @@ export class InstanceSettingsComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit(): void {
-    this.settingsList = this.authService.isAllowedMapper(this.defaultSettingsList, (setting) => setting.requiredRoles.admin);
+    this.settingsList = this.authService.isAllowedMapper(
+      this.defaultSettingsList,
+      (setting) => setting.requiredRoles.admin || [],
```

Comment:
Hi @peintnermax, I've found that if we forget to add required roles to a setting that would cause an error, so if setting.requiredRoles.mgmt is undefined we'll pass an empty array to avoid that